src/Symfony/Component/Serializer/Normalizer/DateIntervalNormalizer.php

```diff
@@ -70,6 +72,27 @@ public function hasCacheableSupportsMethod(): bool
      * @throws UnexpectedValueException
      */
     public function denormalize($data, string $type, string $format = null, array $context = [])
+    {
+        try {
+            $result = $this->doDenormalize($data, $context);
+        } catch (\Throwable $exception) {
```

Comment:
I was wondering why in this case you catch any kind of `Throwable` ?
It seems that so far you were trying to differentiate validation errors from other kinds.
I suppose it's because there is no `NotNormalizableValueException` throw here but then maybe you could either:
* Catch only exception of types `InvalidArgumentException` and `UnexpectedValueException` from the serializer component
* If you want to be less conservative then only exceptions from the serializer component with `ExceptionInterface`

Reply:
Agreed, being as specific as possible is better, I've updated the `catch` clause.